packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -153,6 +153,60 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
       error:error];
 }
 
+// Returns frame rate supported by format closest to targetFrameRate.
+static double bestFrameRateForFormat(AVCaptureDeviceFormat *format, double targetFrameRate) {
+  double bestFrameRate = 0;
+  double minDistance = DBL_MAX;
+  for (AVFrameRateRange *range in format.videoSupportedFrameRateRanges) {
+    double frameRate = MIN(MAX(targetFrameRate, range.minFrameRate), range.maxFrameRate);
+    double distance = fabs(frameRate - targetFrameRate);
+    if (distance < minDistance) {
+      bestFrameRate = frameRate;
+      minDistance = distance;
+    }
+  }
+  return bestFrameRate;
+}
+
+// Finds format with same resolution as current activeFormat for which bestFrameRateForFormat
+// returned frame rate closest to mediaSettings.framesPerSecond. Preferred are formats with the
+// same subtype as current activeFormat. Sets this format as activeFormat and also updates
```

Comment:
How about "Sets this format as the activeFormat in captureDevice and [...]"; I wasn't sure what `activeFormat` was referring to without reading the implementation.